src/aspire/utils/preprocess.py

```diff
@@ -218,27 +142,26 @@ def downsample(insamples, szout, mask=None):
         # stack of one dimension objects
 
         for idata in range(ndata):
-            insamples_fft = crop_pad(fftshift(fft(insamples[:, idata])), L_out)*mask
-            outsamples[:, idata] = np.real(ifft(ifftshift(insamples_fft))*(L_out / L_in))
+            insamples_fft = crop_pad(fftshift(fft(insamples[idata])), L_out)*mask
```

Comment:
Again, part of this is just filtering by a sinc, so we should move this out to an application of a `Filter` object at some point.

Reply:
#256